lnwire/fuzz_test.go

```diff
@@ -264,6 +264,17 @@ func FuzzError(f *testing.F) {
 	})
 }
 
+func FuzzWarning(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		// Prefix with MsgWarning.
+		data = prefixWithMsgType(data, MsgWarning)
+
+		// Pass the message into our general fuzz harness for wire
+		// messages!
```

Comment:
`s/!/.`

Reply:
This is a direct copy-paste from the other fuzz tests in this file.  Seems weird to fix it here and not everywhere else.